script/verification/Verification.s.sol

```diff
@@ -100,16 +101,28 @@ contract SuperchainRegistry is CommonBase {
         proxies.L1CrossDomainMessenger = stdToml.readAddress(toml, "$.addresses.L1CrossDomainMessengerProxy");
         proxies.L1StandardBridge = stdToml.readAddress(toml, "$.addresses.L1StandardBridgeProxy");
         proxies.SystemConfig = stdToml.readAddress(toml, "$.addresses.SystemConfigProxy");
-        proxies.AnchorStateRegistry = stdToml.readAddress(toml, "$.addresses.AnchorStateRegistryProxy");
-        proxies.DisputeGameFactory = stdToml.readAddress(toml, "$.addresses.DisputeGameFactoryProxy");
+
+        // Not all chains have the following values specified in the registry, so we will
+        // set them to the zero address if they are not found.
+        proxies.AnchorStateRegistry = tryReadAddress(toml, "$.addresses.AnchorStateRegistryProxy");
+        proxies.DisputeGameFactory = tryReadAddress(toml, "$.addresses.DisputeGameFactoryProxy");
+        chainConfig.unsafeBlockSigner = tryReadAddress(toml, "$.addresses.UnsafeBlockSigner");
 
         chainConfig.chainId = stdToml.readUint(toml, "$.chain_id");
         chainConfig.systemConfigOwner = stdToml.readAddress(toml, "$.addresses.SystemConfigOwner");
-        chainConfig.unsafeBlockSigner = stdToml.readAddress(toml, "$.addresses.UnsafeBlockSigner");
         chainConfig.batchSubmitter = stdToml.readAddress(toml, "$.addresses.BatchSubmitter");
         chainConfig.batchInbox = stdToml.readAddress(toml, "$.batch_inbox_addr");
     }
 
+    function tryReadAddress(string memory toml, string memory key) internal pure returns (address) {
+        try vmSafe.parseTomlAddress(toml, key) returns (address a) {
+            return a;
+        } catch {
+            console.log("failed to read address for ", key);
+            return address(0);
+        }
+    }
```

Comment:
We can delete this, update our forge-std version to the latest release, and instead use: https://github.com/foundry-rs/forge-std/blob/b5a86914561f38735ef1fc357685de3e7c92dc48/src/StdToml.sol#L148

You can either use it like this:
```solidity
stdToml.readAddressOr(toml, "$.addresses.AnchorStateRegistryProxy", address(0));
```
or like this, similar to LibString
```solidity
using StdToml for string;

// --- snip ---

toml.readAddressOr("$.addresses.AnchorStateRegistryProxy", address(0));
```

Reply:
Nice. [8a770be](https://github.com/ethereum-optimism/superchain-ops/pull/418/commits/8a770be124b6d7de8ee085e31dc7eaac0fa43ba3)